readme.markdown

```diff
@@ -746,6 +746,23 @@ This means that you can safely compile using the compiler when developing your
 software, but you should use Nimble to build the package before publishing it
 to ensure that the dependencies you specified are correct.
 
+To compile code that uses nimble managed packages with `nim`,you need
```

Comment:
This is good but IMO it should be under its own section called "Changing the default Nimble dir" or something.

It should emphasise that you shouldn't need this normally because ``~/.nimble`` is already added to Nim's ``nimblePath`` by default.

Reply:
Yeah, I ran into this problem because I had custom nimble path. I will emphasize this in next revision.